extensions/3rdparty/HasCategory/HasCategory.php

```diff
@@ -94,7 +94,7 @@ function IfhascatPfObj( &$parser, $frame, $args ) {
 	 * Here we find out whether the page is really contained in the category.
 	 * If yes return $then (default: 1) else return $else (default: 0).
 	 */
-	function IfhascatPf( &$parser, $page = '', $category = '', $then = '1', $else = '0' ) {
+	function IfhascatPf( $parser, $page = '', $category = '', $then = '1', $else = '0' ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)